changelog.txt

```diff
@@ -1,5 +1,8 @@
 *** Amazon Pay Changelog ***
 
+= 2.0.4 - 2021-xx-xx =
+* Fix - Interference when subcsriptions' payment changes to other payment method.
```

Comment:
Not sure but probably one word is missing:
```suggestion
* Fix - Interference when subcsriptions' payment method changes to other payment method.
```

Reply:
Didn't think of that but nice addition. Thanks!

Reply:
Updated readme as well 460b9ca.